stack/run.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:jammy
+
+ARG sources
+ARG packages
+
+RUN echo "$sources" > /etc/apt/sources.list
+
+RUN echo "debconf debconf/frontend select noninteractive" | debconf-set-selections && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt-get -y $package_args update && \
+  apt-get -y $package_args upgrade && \
+  apt-get -y $package_args install locales && \
+  locale-gen en_US.UTF-8 && \
+  update-locale LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 && \
+  apt-get -y $package_args install $packages && \
+  rm -rf /var/lib/apt/lists/* /tmp/*
```

Comment:
In bionic, we remove a few other things:
https://github.com/paketo-buildpacks/stacks/blob/c00defb718858eb3724c71addae362c9c047f068/bionic/dockerfile/run/Dockerfile#L17-L21

Should we also be removing those here?